### PR TITLE
New. SFW. CustomDB implemented.

### PR DESCRIPTION
**Pull request by svfcode**

### CODE REVIEW

File: cleantalk/fw_nets_meta.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      'columns' => 
+      array (
+        0 => 'network',
+      ),
+      'status' => 'ready',
+      'type' => 'btree',
+    ),
+  ),
+  'cols_num' => 4,
+  'rows' => 13350,
```
Comment (Glomberg) [inline cleantalk/fw_nets_meta.php]: The `rows` must be 0 in the repository.

File: cleantalk/lib/Cleantalk/ApbctUni/File/BTree.php [hunk 2]
```diff
@@ -0,0 +1,388 @@
...
+     * @param mixed $key to search for
+     * @param int $link to the supposed leaf with an element
+     *
+     * @return false|array of BTreeLeafNode
+     */
+    public function get( $key, $link = 0 ){
+        
+        $out = array();
+
+						// $myfile = fopen(__DIR__ . "/btree-get-a.txt", "a") or die("Unable to open file!");
```
Comment (Glomberg) [inline cleantalk/lib/Cleantalk/ApbctUni/File/BTree.php]: Debug?

**Review by Glomberg — CHANGES_REQUESTED** [review]

File: cleantalk/fw_nets_meta.php (hunk 1 above)
Comment (svfcode) [reply cleantalk/fw_nets_meta.php]: Fixed.

File: cleantalk/lib/Cleantalk/ApbctUni/File/BTree.php (hunk 2 above)
Comment (svfcode) [reply cleantalk/lib/Cleantalk/ApbctUni/File/BTree.php]: Yeah, removed, thanks.

**Review by alexandergull — APPROVED** [review]

**Review by Glomberg — APPROVED** [review]
There is one major note - why the library was implemented by simple copy-paste, but not by composer. Wrong way.

**Review by ArtemAnoshin — APPROVED** [review]

**Pull request merged**